config/routes.rb

```diff
@@ -36,6 +36,9 @@
   end
 
   # ***** COURSES AND LESSONS ROUTES *****
+  # deprecated /courses/curriculum route redirect
+  get '/courses/curriculum' => redirect('/courses')
```

Comment:
Ah I see, the courses resource is overriding it. No problem, I'll have a look at restructuring the routes file in a future revision.